docs/guides/bazzite.md

```
@@ -0,0 +1,71 @@
+> A friendly guide for setting up Bazzite on ASUS laptops
+
+Newcomers should start by reading the [Intro](../introduction.md) guide.
+
+Bazzite is a gaming-oriented atomic Fedora image based on [Universal Blue](https://universal-blue.org/). Like other atomic Fedora images it uses rpm-ostree, and `asusctl` is not preinstalled. Bazzite ships Homebrew preconfigured and the Terra repository preconfigured but disabled, which gives you two ways to install asusctl.
+
+## Contents
+
+- [Recommended: ujust asus](#recommended-ujust-asus)
+- [Alternative: Terra repository](#alternative-terra-repository)
+- [Graphics Switching](#graphics-switching)
+- [ROG Ally and Ally X](#rog-ally-and-ally-x)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)